guides/deployment/deployment.md

```diff
@@ -65,7 +65,7 @@ This step is required only if you have static assets like images, JavaScript, st
 Compilation of static assets happens in two steps:
 
 ```console
-$ cd assets && webpack --mode production
+$ cd assets && npm run deploy && cd -
```

Comment:
By using npm, we can simply use the prefix option instead of cd in the folder and cd - after 😉

Reply:
@simonprev Nice suggestion!